### PR TITLE
[AF-1128] [Patch] Make shallow copy of window instead of deep

### PR DESCRIPTION
**Pull request by Mehul-Porwal**
Making a shallow copy of window as window object can have references to other iframes, and the browser will not allow us deep copy an iframe from a different origin.

Error in browser without this fix:
image

@zendesk/vegemite

### CODE REVIEW

**Comment by keitheous** [comment]
👍 true~

File: lib/zendesk_apps_support/assets/installed.js.erb [hunk 1]
```diff
@@ -1,33 +1,51 @@
...
-  wrappedWindow.top = wrappedWindow;
+  <% if apps_zaf_naughty_v1_logging %>
+    var self = this;
+    var reporter = window.ZendeskReporter ? new window.ZendeskReporter() : {};
+
+    function getWrappedWindow(appName) {
+      // This is just defining a wrapped version of jQuery which simply makes a
+      // pass through call to global window.jQuery and adds logging information
+      // about the origin of the page to Datadog.
+      function wrapped$(appName) {
```
Comment (hoangtrvu) [inline lib/zendesk_apps_support/assets/installed.js.erb]: `appName` should not be here

File: lib/zendesk_apps_support/assets/installed.js.erb [hunk 2]
```diff
@@ -1,33 +1,51 @@
...
+        var tags = ['origin:' + self.location.hostname];
+        if (this.__app_name) {
```
Comment (hoangtrvu) [inline lib/zendesk_apps_support/assets/installed.js.erb]: How does `this.__app_name` work here? Can we just use `appName` directly?

Comment (Mehul-Porwal) [reply lib/zendesk_apps_support/assets/installed.js.erb]: Yes, this is leftover from some previous code change, forgot to remove it.

File: lib/zendesk_apps_support/assets/installed.js.erb (hunk 1 above)
Comment (Mehul-Porwal) [reply lib/zendesk_apps_support/assets/installed.js.erb]: 👍

File: lib/zendesk_apps_support/assets/installed.js.erb (hunk 2 above)
Comment (Mehul-Porwal) [reply lib/zendesk_apps_support/assets/installed.js.erb]: Removed it! 👍

**Pull request merged**